core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ErrorReportable.scala

```diff
@@ -1,23 +1,22 @@
 package org.broadinstitute.dsde.rawls.dataaccess
 
 import com.google.api.client.googleapis.json.GoogleJsonResponseException
-import org.broadinstitute.dsde.rawls.model.ErrorReport
+import org.broadinstitute.dsde.rawls.model.{ErrorReport, ErrorReportSource}
 import spray.http.StatusCodes
 import spray.httpx.UnsuccessfulResponseException
 
 trait ErrorReportable {
-  val errorReportSource: String
+  def errorReportSource: ErrorReportSource
```

Comment:
I had extenders of this trait define their own implicit errorReportSource, but the implicit lookup-ness was finding the one in dsde.rawls.model's package.scala. Making it be abstract in this class and then used explicitly solved that problem.